tests/LineScanCameraTests.cpp

```diff
@@ -46,87 +46,62 @@ TEST_F(ConstVelocityLineScanSensorModel, Inversion) {
   EXPECT_DOUBLE_EQ(imagePt.samp, imageReprojPt.samp);
 }
 
-TEST_F(ConstVelocityLineScanSensorModel, OffBody1) {
+TEST_F(ConstVelocityLineScanSensorModel, OffBody) {
    csm::ImageCoord imagePt(4.5, 4.0);
    csm::EcefCoord groundPt = sensorModel->imageToGround(imagePt, 0.0);
    EXPECT_NEAR(groundPt.x, 0.063995905, 1e-8);
    EXPECT_NEAR(groundPt.y, -7.999488033, 1e-8);
    EXPECT_NEAR(groundPt.z, 8, 1e-8);
 }
-TEST_F(ConstVelocityLineScanSensorModel, OffBody2) {
-   csm::ImageCoord imagePt(4.5, 12.0);
-   csm::EcefCoord groundPt = sensorModel->imageToGround(imagePt, 0.0);
-   // EXPECT_NEAR(groundPt.x, 0.44979759, 1e-8);
-   // EXPECT_NEAR(groundPt.y, 14.99325304, 1e-8);
-   // EXPECT_NEAR(groundPt.z, -14.99325304, 1e-8);
-}
 
-TEST_F(ConstVelocityLineScanSensorModel, OffBody3) {
-   csm::ImageCoord imagePt(12.5, 4.0);
-   csm::EcefCoord groundPt = sensorModel->imageToGround(imagePt, 0.0);
-   // EXPECT_NEAR(groundPt.x, 0.44979759, 1e-8);
-   // EXPECT_NEAR(groundPt.y, 14.99325304, 1e-8);
-   // EXPECT_NEAR(groundPt.z, 14.99325304, 1e-8);
+TEST_F(ConstVelocityLineScanSensorModel, ProximateImageLocus) {
+   csm::ImageCoord imagePt(8.5, 8.0);
```

Comment:
For this test case and this image point, the image ray is f(t) = (1000, 0, 0) + t*(-1, 0, 0)